reportportal_client/items/rp_base_item.py

```diff
@@ -23,25 +23,44 @@ class BaseRPItem(object):
     """This model stores attributes related to RP item."""
 
     def __init__(self, rp_url, session, api_version, project_name,
-                 launch_uuid):
+                 launch_uuid, generated_id):
         """Initialize instance attributes.
 
         :param rp_url:         report portal url
         :param session:        Session object
         :param api_version:    RP API version
         :param project_name:   RP project name
         :param launch_uuid:    Parent launch UUID
+        :param generated_id:   Id generated to speed up client
         """
         self.uuid = None
         self.weight = None
+        self.generated_id = generated_id
         self.http_requests = list()
-        self._response = None
+        self._responses = []
```

Comment:
Currently, there are two ways of allocating lists, here as [] and above as list().
Let's stick to a single approach.

JFYI, [] is faster
In [1]: timeit []
17.1 ns ± 0.465 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)
In [2]: timeit list()
61.1 ns ± 2.97 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

Reply:
Fixed